### PR TITLE
Fixes #20263: Missing SELinux rules for httpd on RockyLinux

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/20263

### CODE REVIEW

File: rudder-server-relay/SPECS/rudder-server-relay.spec [hunk 1]
```diff
@@ -75,7 +75,7 @@ BuildRequires: selinux-policy-devel
 %endif
 
 %if 0%{?rhel} && 0%{?rhel} > 7
-Requires: python3-policycoreutils
+Requires: python3-policycoreutils policycoreutils-python-utils
```
Comment (VinceMacBuche) [inline rudder-server-relay/SPECS/rudder-server-relay.spec:78]: This package exist on all rhel8 ?

Comment (amousset) [reply rudder-server-relay/SPECS/rudder-server-relay.spec:78]: yes

**Review by Fdall — APPROVED** [review]

**Comment by amousset** [comment]
OK, merging this PR

**Comment by amousset** [comment]
OK, merging this PR

**Pull request merged**